README.md

```diff
@@ -29,8 +29,7 @@ In a similar fashion as namespaces, you can create AWS resources in your desired
 
 ### Changes within namespaces
 
-Changes within namespaces directory are managed by concourse job configured with `fly` and running on live-0 cluster.
-Command used to create build job is `fly --target live0 sp -c pipeline.yaml -p build-environments`.
+Changes within namespaces directory are managed by the `build-environments` concourse job configured [here](https://github.com/ministryofjustice/cloud-platform-concourse/tree/master/pipelines/cloud-platform-live-0/main/build-environments.yaml).
```

Comment:
Link is broken.

Reply:
pending https://github.com/ministryofjustice/cloud-platform-concourse/pull/8

Reply:
The above is now merged and the link is working now. Merging this one as well.